content/reference/sql/sql-support/miscellaneous.md

```diff
@@ -113,6 +111,74 @@ select `index`, `position`, row_count, distinct_count, columns, upper_bound, upp
 +---------+----------+-----------+----------------+----------+-------------+-----------------+-----------+
 ```
 
+### Auto-Refresh
+
+Static statistics become stale quickly for tables that change frequently. Users can choose to manually manage run `ANALYZE` statements, or use some form of auto-refresh.
+
+Auto-refresh statistics work the same way as partial `ANALYZE` updates. A table's "former" and "new" chunk set will 1) share common chunks preexisting in "former" 2) differ by deleted chunks only in the "former" table, and 3) differ by new chunks in the "new" table. This mirror's Dolt's inherent structural sharing. Rather than writing a new set of statistics for on every refresh interval (like an `ANALYZE` call does), we can toggle how many changes triggers the update.
```

Comment:
> Rather than writing a new set of statistics for on every refresh interval (like an `ANALYZE` call does)

I don't follow this. An ANALYZE call doesn't have a refresh interval, does it?